gammapy/stats/variability.py

```diff
@@ -82,3 +84,76 @@ def compute_chisq(flux):
     yobs = flux.data
     chi2, pval = stats.chisquare(yobs, yexp)
     return chi2, pval
+
+
+def compute_dtime(flux, flux_err, time, axis=0):
+    r"""Calculate the characteristic doubling time for a series of measurements.
+
+    The characteristic doubling time  is estimated to obtain the
+    minimum variability timescale for the light curves in which
+    rapid variations are clearly evident: for example it is useful in AGN flaring episodes.
+
+    This quantity, especially for AGN flares, is often expressed
+    as the pair of doubling time and halving time, or the minimum characteristic time
+    for the rising and falling components respectively.
+
+    Parameters
+    ----------
+    flux : `~astropy.units.Quantity`
+        the measured fluxes
+    flux_err : `~astropy.units.Quantity`
+        the error on measured fluxes
+    time: `~astropy.units.Quantity`
+        the times at which the fluxes are measured
+    axis : int, optional
+        Axis along which the value is computed.
+        The default is to compute the value on axis 0.
+
+    Returns
+    -------
+    dtime, dtime_err : `~numpy.ndarray`
+        Characteristic doubling time, halving time and errors.
+
+    References
```

Comment:
Since users will be more exposed to the wrapper function, this detailed doc string can lie there..